spec/models/user_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user){create(:user)}
+  #let(:user) { User.new(email: 'user1@example.com', password: 'password', password_confirmation: 'password') }
```

Comment:
kindly remove comment from your code, you can always use git history to get previous code right :slightly_smiling_face: